README.md

```diff
@@ -213,12 +213,24 @@ libController.on("itemMouseLeave", function (searchText) {
 The string property for the event name is: ItemMouseLeaveEventName. So the following achieves the same:
 
 ```js
-libController.on(libController.ItemMouseLeaveEventName, function(contextData) {
+libController.on(libController.ItemMouseLeaveEventName, function(arg) {
     console.log("Data: " + arg.data);
     console.log("Rect(top, left, bottom, right): " + arg.rect.top + "," + arg.rect.left + "," + arg.rect.bottom + "," + arg.rect.right);
 })
 ```
 
+### Event 'itemSummaryExpand'
```

Comment:
Can we call it `itemSummaryExpanded`? It aligns with all other events (except probably *Enter* and *Leave*, which are somewhat okay because of historical reasons, but we can change those too next time).